both.ts

```diff
@@ -0,0 +1,23 @@
+import curryN from "./utils/curry_n.ts"
+import { Func, Curry2 } from "./utils/types.ts"
+import { isFunction } from "./utils/is.ts"
+import { lift } from "./lift.ts"
+import { and } from "./and.ts"
+
+function _both(f: Func, g: Func) {
+  return isFunction(f) ?
```

Comment:
don't use ternary operator if either condition is long or the code blocks contain more than 2 lines